modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -535,12 +535,17 @@ definition {
 	}
 
 	@description = "LPS-129928 - Verify that when the user makes changes to the Process Builder these changes are reflected in the XML code"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test ChangesOnDiagramAreReflectedOnXML {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#ChangesOnDiagramAreReflectedOnXML";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "Workflow Title");
 
-		// TODO LRQA-73000 ChangesOnDiagramAreReflectedOnXML pending implementation
+		ProcessBuilderKaleoDesignerReact.gotoSourceOrDiagram();
 
+		ProcessBuilderKaleoDesignerReact.viewChangesReflectsCodeXML(changesCode = "Workflow Title");
 	}
```

Comment:
At this test, we undesteded that the Definition title not necesserally will be present at the XML, so it doesn't make sense keep our coverage based on that, so i have added some more steps like edit a node label by the diagram and assert that in the source view it's reflected. Please see how it was sent and merged at the respective file. 
For references: [pull/267](https://github.com/lucasperj/liferay-portal/pull/267/files)